comparators/invalid_tag_modification.js

```diff
@@ -22,10 +22,14 @@ function invalidTagModification(newVersion, oldVersion, callback) {
   var result = {};
   if (!newVersion || !oldVersion) return callback(null, result);
```

Comment:
Now we assume that newVersion will always be present in a geojson. And if a feature is deleted, we will check for `deleted` tag of new version. Ref commit : https://github.com/mapbox/osm-compare/commit/2e0e6b68a5e5131589b6cac69b9b1945dbf98ed6

So `(!newVersion || !oldVersion)` will become `(newVersion.deleted || !oldVersion)`